src/indexing.jl

```diff
@@ -369,4 +369,11 @@ Base.checkindex(B::Type{Bool}, inds::AbstractUnitRange, i::StaticIndexing{T}) wh
 
 # unsafe_view
 
-Base.unsafe_view(A::AbstractArray, i::StaticIndexing{T}) where T = Base.unsafe_view(A, unwrap(i))
+unwrap_if_needed(i::StaticIndexing) = unwrap(i)
+unwrap_if_needed(i::Base.ViewIndex) = i
```

Comment:
It looks like you could just add a method to `unwrap` itself rather than having `unwrap_if_needed`.

Reply:
`unwrap` has only one method that works on `StaticIndexing`, should I add this?
```julia
unwrap(i::Base.ViewIndex) = i
```